ext/standard/basic_functions.stub.php

```diff
@@ -2467,8 +2567,14 @@ function strnatcmp(string $string1, string $string2): int {}
  */
 function localeconv(): array {}
 
+/**
+ * @compile-time-eval
+ */
 function strnatcasecmp(string $string1, string $string2): int {}
```

Comment:
is this function locale safe?

Reply:
No

Reply:
Thank you. I will remove. Can you please recheck the other functions?